pkg/kubelet/kubelet.go

```diff
@@ -1603,26 +1599,24 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	podAdmitSpan.End()
-	checkNetworkCtx, networkConditionSpan := trace.StartSpan(podAdmitCtx, "Network condition check")
 
 	// If the network plugin is not ready, only start the pod if it uses the host network
 	if rs := kl.runtimeState.networkErrors(); len(rs) != 0 && !kubecontainer.IsHostNetworkPod(pod) {
+		_, networkConditionSpan := trace.StartSpan(ctx, "Network condition check")
```

Comment:
This is just tracing the creation of the kubernetes event, which probably isn't useful.  I would probably remove this altogether, or put the tracing within the `networkErrors()` function.

Reply:
or rather, it doesn't need to be in the function, but around it here.